bin/sumd

```diff
@@ -163,6 +170,11 @@ def main():
     if not options.foreground:
         log.info("Creating pidfile with PID: %s" % (pidfile(create=True)))
 
+    if options.suspend:
```

Comment:
Could you share some context about the problem this is trying to solve? I don't necessarily object to this change, but I don't yet understand the use case. 

For example, in the default configuration of sumd, and our Riemann config, the TTLs are set such that a delay here could cause you to fall 'outside' that TTLs and the event to expire. I also believe (but would have to doublecheck) that events are scheduled for now+interval on startup, so that does provide some natural delay already.